### DISCUSSION TITLE
MMKV不可用后，我们可以做些什么？

### DISCUSSION THREAD

**Discussion by BannaGo** [Tencent/MMKV#1079] [General]
1、mmkv不可用
我注意到有个接口：- isFileValid:(NSString *)mmapID 来判断mmkv是否可用。
那如果判断出该mmkv不可用时，是不是意味着我们永远丢失了这个mmkv文件里的数据？
在这种情况下，调用[mmkv getObject/set] 这类接口会发生什么？

2、怎么办？
判断出不可用后，我们能做些什么了？

**Comment by lingol** [comment]
You don't need to do anything.
